NDB.Covid19/NDB.Covid19.Droid/Services/DroidDialogService.cs

```diff
@@ -10,7 +10,7 @@ public class DroidDialogService : IDialogService
         public void ShowMessageDialog(string title, string message, string okBtn, PlatformDialogServiceArguments platformArguments = null)
         {
             Activity activity = platformArguments?.Context != null && platformArguments.Context is Activity act ? act : Current.Activity;
-            DialogUtils.DisplayDialog(activity, title, message, okBtn);
+            DialogUtils.DisplayDialogAsync(activity, title, message, okBtn);
```

Comment:
Deleted the method. The async part of this is just for getting the result of the dialog. Do you think it will be issues when we don't use that result here?

Reply:
no, the result will not be a problem. I am worried about async behavior that it do something to early.

Reply:
I think it should be fine then, apart from the result the two methods are practically identical